assets/templates/command/install_server.md

```diff
@@ -0,0 +1,20 @@
+{{ .ConfluenceURL }} has been successfully added. To finish the configuration, add an Application Link in your Confluence instance following these steps:
+
+1. Navigate to [**Settings > Applications > Application
+   Links**]({{ .ConfluenceURL }}/plugins/servlet/applinks/listApplicationLinks)
+2. Click **Create link**.
+3. In **Create Link** screen, select **External Application** and **Incoming** as
+   `Application type` and `Direction` respectively. Click **Continue**.
+4. In **Link Applications** screen, set the following values:
+**Name**: `Mattermost`
+**Redirect URL**: ```{{ .RedirectURL }}```.
+**Application Permissions**: `Admin`
+Click **Continue**
+5. Copy the `clientID` and `clientSecret` and add them into the plugin configuration.
+6. Use the "/confluence connect" command to connect your Mattermost account with your
+   Confluence account.
+7. Use the "/confluence connect URL admin" for confluence admin only to save the token for the first time only.   
```

Comment:
is this a slash command? Is it a one-time command?
is it available only to Confluence admins?

Reply:
yes this a slash command (This is just a slight variant of /confluence connect command) and this should be used only by the confluence admins for the first time as to create webhooks for confluence server we required admin permissions and token so with this command we securely save the admin token just for the creation and deletion of the webhooks for confluence server for every user